tekton/publish.yaml

```diff
@@ -144,6 +144,8 @@ spec:
 
       # Publish images and create release.yaml
       ko resolve --preserve-import-paths -t $(inputs.params.versionTag) -f /workspace/go/src/github.com/tektoncd/pipeline/config/ > /workspace/output/bucket/latest/release.yaml
+      # Publish images and create release.notags.yaml
```

Comment:
this comment might be a good place to indicate _why_ we need a release.notags.yaml?

Reply:
Very good point :+1: